src/schema/test_schema.py

```diff
@@ -38,15 +38,15 @@ def test_messages_from_langchain():
         assert str(e) == "Unsupported message type: SystemMessage"
 
 
-def test_message_run_id_usage():
+def test_message_run_id_usage() -> None:
     run_id = "847c6285-8fc9-4560-a83f-4e6285809254"
     lc_message = AIMessage(content="Hello, world!")
     ai_message = ChatMessage.from_langchain(lc_message)
     ai_message.run_id = run_id
     assert ai_message.run_id == run_id
 
 
-def test_messages_tool_calls():
+def test_messages_tool_calls() -> None:
```

Comment:
Is there a benefit to doing this for tests? It isn’t a common practice as far as I know

Reply:
The main issue is that ruff and mypy will complain if they don't have a return type annotation (if you have it set).

Of course, I agree that for tests, these return type hints aren't particularly useful. If you think it's better not to annotate in tests, I can change this part back.

Reply:
Ah OK I see even in Langchain itself they do this. I hadn't seen that before in a popular library (of course I don't always go spelunking through the tests)

https://github.com/langchain-ai/langchain/blob/master/libs/core/tests/unit_tests/messages/test_utils.py

I'm fine with it then. TY!